README.md

```diff
@@ -1 +1,143 @@
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Rails React <a name="about-project"></a>
+
+Hello Rails React is an tutorial for react and rails in a single project.
+
+## 🛠 Built With <a name="built-with"></a>
+This app is build with Ruby on Rails and Postgresql.
+### Tech Stack <a name="tech-stack"></a>
+> <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+> <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+> <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+> <li><a href="https://reactjs.org/">React</a></li>
+> <li><a href="https://redux.js.org/">Redux</a></li>
+### Key Features <a name="key-features"></a>
+
+> - User can create a See a random Greeting
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+## To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Mac or PC
+- Install Ruby
+- Install Rails
+- Install Postgresql
+- Install Yarn
+- Understanding of Ruby
+
+### Setup
+
+- Clone this repository to your desired folder:
+
+```sh
+ git clone https://github.com/gauravgangwarweb/hello-rails-react.git
+```
+
+- Navigate into the cloned folder
+
+```sh
+ cd hello-rails-react
+```
+### Install
+
+- Install this project with:
+
+```sh
+ bundle install
+ yarn install
+```
+
+- Set up the database
+  Change the username and password of your postgres account in `config/database.yml`
+
+```sh
+ rails db:create db:migrate db:seed
+```
+
+- Start the server
+
+```sh
+ ./bin/dev
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gaurav Gangwar**
+
+- GitHub: [@Gaurav](https://github.com/gauravgangwarweb/)
+- LinkedIn: [@Gaurav](https://www.linkedin.com/in/gauravgangwarweb/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/gauravgangwarweb/hello-rails-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- I would like to thank Microverse for providing us with reading materials that aided us to during the project development
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/gauravgangwarweb/hello-rails-react/blob/app/MIT.md) licensed.
+
```

Comment:
- [ ] Please make sure that you fix the link to your license file. Currently, it is a broken link :pray: